src/redux/books/books.js

```diff
@@ -0,0 +1,37 @@
+const addingBooks = 'bookstore/books/addingBooks';
+const removingBooks = 'bookstore/books/removingBooks';
+const initialState = { bookList: [] };
+
+export default function bookReducer(state = initialState, action) {
+  switch (action.type) {
+    case addingBooks:
+      return [
+        ...state,
+        {
+          title: action.title,
+          author: action.author,
+          id: action.id,
+        },
+      ];
+    case removingBooks:
+      return state.bookList.filter((book) => book.id !== action.id);
+    default:
+      return state;
+  }
+}
+
+export function addBook(Title, Author, Id) {
+  return {
+    type: addingBooks,
+    title: Title,
+    author: Author,
+    id: Id,
+  };
```

Comment:
- [ ] Kindly make sure to fix add, remove function as I am not able to add or delete a book.
- [ ] Also kindly make sure that the check status button work as expected, it should display "under construction" upon clicking.

https://user-images.githubusercontent.com/70269507/186594763-61abbb82-6f62-47a9-abf6-2945b2c719cc.mov